src/test/java/Homework21Test.java

```diff
@@ -0,0 +1,34 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework21Test extends BaseTest{
+
+    @Test
+    public void renamePlaylist () {
+
+
+        String newNamePlaylist ="Summer songs";
+        login("marianna.tysovska@gmail.com", "marysia288940.");
+        doubleClickChoosePlaylist();
```

Comment:
Please, add Assert in test

HINT

   **Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```


**One test - One Assert**